### PR TITLE
Updated script.sh

### PR DESCRIPTION
**Pull request by helloakash1701**
Improvements made in this version:

1- Added informative echo statements to provide feedback during the execution of the script.

2 -Assigned the file name and target directory to variables (zip_file and target_dir, respectively) for easier customization and maintenance.

3- Added an error check after the zip operation (zip -r "$zip_file" "$target_dir") using $? to determine the exit status of the previous command. If the exit status is zero, it indicates success, and a corresponding message is displayed. Otherwise, a failure message is shown.

These modifications enhance the script by providing clearer output and making it more flexible for customization.

**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 699cf4bacf67fa3445a631dbabbcf773e3dbb0e3 at: https://6492adf65fc3a229b060d4e9--meshery-docs-preview.netlify.app

**Comment by helloakash1701** [comment]
> 🚀 Preview for commit [699cf4b](https://github.com/meshery/meshery/commit/699cf4bacf67fa3445a631dbabbcf773e3dbb0e3) at: https://6492adf65fc3a229b060d4e9--meshery-docs-preview.netlify.app

sorry what?

**Comment by Yashsharma1911** [comment]
@helloakash1701 
Let's discuss this on Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)

**Comment by helloakash1701** [comment]
> @helloakash1701 
> Let's discuss this on Meshery Dev call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit#) if you would. :)
Sure :)

**Comment by leecalcote** [comment]
Thanks for this, @helloakash1701.

@hexxdump @alphaX86 @abdullah1308 @suhail34 are these improvements desirable?

File: docs/script.sh [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
+# Create a new zip file
+echo "Creating $zip_file..."
+zip -r "$zip_file" "$target_dir"
+
+# Check the zip operation result
+if [ $? -eq 0 ]; then
+    echo "Zip file created successfully."
+else
+    echo "Failed to create zip file."
 fi
```
Comment (hexxdump) [inline docs/script.sh:21]:
```suggestion
if [ $? -ne 0 ]; then
    echo "Failed to create zip file."
fi
```

File: docs/script.sh [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+
+# Create a new zip file
+echo "Creating $zip_file..."
+zip -r "$zip_file" "$target_dir"
+
+# Check the zip operation result
+if [ $? -eq 0 ]; then
+    echo "Zip file created successfully."
+else
+    echo "Failed to create zip file."
```
Comment (hexxdump) [inline docs/script.sh:20]: if the zip file is not created successfully, the install script should exit at this point.

**Comment by alphaX86** [comment]
@helloakash1701 there are some feedbacks to address on this PR...

**Pull request closed without merging**